src/version.h

```diff
@@ -33,4 +33,9 @@ static const int MEMPOOL_GD_VERSION = 60002;
 //! "filter*" commands are disabled without NODE_BLOOM after and including this version
 static const int NO_BLOOM_VERSION = 170004;
 
+//! Changes to CInv parsing, starting with this version:
+//! - MSG_WTX type defined, which contains two 32-byte hashes.
+//! - Undefined message types now return an error instead of being ignored.
```

Comment:
This should be documented in ZIP 239; I will do that.

Reply:
Likewise, this bullet-point should be part of the later commit; the PR currently does not implement this.